opencensus/stats/measure_to_view_map.py

```diff
@@ -55,12 +55,21 @@ def get_view(self, view_name, timestamp):
 
         view_data_list = self._measure_to_view_data_list_map.get(
             view.measure.name)
-        if view_data_list is not None:
-            for view_data in view_data_list:
-                if view_data.view.name == view_name:
-                    view_data_copy = copy.deepcopy(view_data)
-                    view_data_copy.end()
-                    return view_data_copy
+
+        if not view_data_list:
+            return None
+
+        for view_data in view_data_list:
+            if view_data.view.name == view_name:
+                break
+        else:
+            return None
```

Comment:
I rewrote this for clarity, the only behavior change is the copy below. Note that we're not using `timestamp` here, there are likely other bugs hiding in this class.